README.md

```diff
@@ -1,68 +1,51 @@
-# EVE
 
-This project aims to simulate basic lifeforms evolving and getting smarter, in a simplified manner.
+# EVE
 
-## Development
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-### Setup
+##### Simulate simple, artificial lifeforms as they learn and evolve
 
-1. Install cmake & [Catch2](https://github.com/catchorg/Catch2) on your OS
-2. Clone this repository:  
-   HTTPS: `git clone https://github.com/Xerus2000/eve`  
-   SSH: `git clone git@github.com:Xerus2000/eve.git`
-3. Build 'eve' via CLion so that all tasks become available for running
-4. In CLion go to `Settings > Editor > File and Code Templates` and remove the "Created by" header in c/cpp files
-5. Execute this to set up the git commit message hooks:  
-   `git config core.hooksPath .dev/githooks`
+----
 
-### Structure
+#### Table of Content
 
-All production code goes into [src](src), separated into [model](src/model), [view](src/view) and [controller](src/controller).  
-[test](test) contains the tests and mirrors the source file structure.
+1. Setup
+2. Description
+3. Collaboration
+4. About
 
-### Classes and Inheritance
+----
 
-The folder structure should represent classes and inheritance.
+### Setup
 
-Therefore,
+* **Requirements:** Build tools: `cmake`, `make`, an ISO C++17 Compiler
+* **Download**: ``` git clone https://github.com/Xerus2000/eve```
+* **Build:** Run `cmake` and then `make`
+* **Run:** Start the simulation by executing `eve`
 
-- every class belongs in its own folder with the same name
-- every sub-class belongs within its parents folder
+----
 
-Thus, as of currently we have the following directory tree:
+### Description
```

Comment:
shouldnt be a separate section, this should come _directly_ below the headline before the table of contents